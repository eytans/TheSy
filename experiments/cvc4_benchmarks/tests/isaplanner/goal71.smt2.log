19:25:20 [WARN] Collecting statistics
19:25:20 [WARN]  ex is: zero
19:25:20 [WARN]  ex is: (succ zero)
19:25:20 [WARN]  ex is: (succ (succ zero))
19:25:20 [WARN] Running exploration without proof mode, but goals were given
19:25:20 [WARN] Proof mode: true
19:25:20 [WARN]  ex is: zero
19:25:20 [WARN]  ex is: (succ zero)
19:25:20 [WARN]  ex is: (succ (succ zero))
19:25:20 [WARN] Running exploration without proof mode, but goals were given
19:25:20 [WARN] Proof mode: true
19:25:20 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:25:20 [WARN] proved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:25:21 [WARN] proved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:25:21 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:25:21 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0))
19:25:21 [WARN] proved: (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => (leq (succ zero) (succ ?ts_ph_Nat_0))
