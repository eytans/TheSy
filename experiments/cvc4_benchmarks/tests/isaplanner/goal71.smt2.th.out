19:25:20 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(less ?x zero) => false
(less zero (succ ?x)) => true
(less (succ ?x) (succ ?y)) => (less ?x ?y)
(less ?x ?y) => (less (succ ?x) (succ ?y))
Searcher: (less ?autovar5 (succ ?autovar1))
Searcher: (less ?autovar5 (succ ?autovar3))
Searcher: (less zero ?autovar6)
Searcher: (less (succ ?autovar4) ?autovar6)
19:25:20 [0m[33m[WARN] [0m ex is: zero
19:25:20 [0m[33m[WARN] [0m ex is: (succ zero)
19:25:20 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:25:20 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:25:20 [0m[33m[WARN] [0mProof mode: true
19:25:20 [0m[33m[WARN] [0m ex is: zero
19:25:20 [0m[33m[WARN] [0m ex is: (succ zero)
19:25:20 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:25:20 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:25:20 [0m[33m[WARN] [0mProof mode: true
19:25:20 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:25:20 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:25:21 [0m[33m[WARN] [0mproved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:25:21 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:25:21 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0))
19:25:21 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 (succ ?ts_ph_Nat_0)) => (leq (succ zero) (succ ?ts_ph_Nat_0))
done in 3954
